.dockerignore

```diff
@@ -0,0 +1 @@
+target/
```

Comment:
I wonder if it's necessary to keep this under revision.

Reply:
this is used because if you don't have that docker builds take a long time because it tries to add the whole `target` folder into Docker build context and in my machine it's around 7GB

Reply:
Thanks for the explanation, I wasn't aware of that!